packages/frontend/src/config/urls.ts

```diff
@@ -71,6 +70,12 @@ export const TILE_PATTERN = '/tiles/:tileId'
 export const PUBLIC_TILE_PATTERN = '/tiles/:tileId/:viewOnlyKey'
 export const UNAUTHORIZED_TILE = '/tiles/unauthorized'
 
+// flow editor: default to flows if flowid is undefined
+export const FLOW_EDITOR = (flowId: string | undefined): string =>
```

Comment:
nit: convention is to use `?` for undefined, but I can also see why you did it this way (it makes it clearer that an undefined id is the edge case).

Up to you.

```suggestion
export const FLOW_EDITOR = (flowId?: string): string =>
```

Reply:
ok yep my thought was as u mentioned, ill leave it in the comment instead, thanks!